bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,37 @@
+{% extends "bare.html" %}
+{% stylesheet 'firefox_new_firstrun' %}
+{% block site_js %}
+  {% javascript 'site' %}
+  {% javascript 'common' %}
+  {% javascript 'firefox_new_firstrun' %}
+{% endblock %}
+
+{% block content %}
+  <div>
+    <div id="scene">
+      <div class="fxaccounts-container">
+        <h1 id="title">{{_('Already Using Firefox?')}}</h1>
+        <div id="left-divider">
+          <p>{{_('Sign in to your account and we\'ll sync the bookmarks, passwords and other great things you\'ve saved to Firefox on other devices.')}}</p>
```

Comment:
Nit: for consistency with other page copy on mozorg you can use curly quotes `’` here instead of escaping.